### ISSUE TITLE
minCSS Option Throws Error in 2.17.2 and Up

### ISSUE THREAD

**Issue by smschrader** [dowjones/gulp-bundle-assets#50]
Error - 
```javascript
events.js:74
        throw TypeError('Uncaught, unspecified "error" event.');
...
  at ...\node_modules\gulp-bundle-assets\n
ode_modules\gulp-minify-css\node_modules\clean-css\lib\clean.js:71:41
  at process._tickCallback (node.js:419:13)
```

is thrown when option is enabled for Less files. Issues goes away with earlier releases. 
Seems to be an issue with the clean-css module. 

Might be related to Issue https://github.com/jonathanepollack/gulp-minify-css/issues/61

Sample Config:
```javascript
var lazypipe = require(__base + 'lazypipe'),
    gif = require(__base + 'gulp-if'),
    less = require(__base + 'gulp-less');

function stringEndsWith(str, suffix) {
  return str.indexOf(suffix, str.length - suffix.length) !== -1;
}

function isLessFile(file) {
  return stringEndsWith(file.relative, 'less');
}

var styleTransforms = lazypipe()  
  .pipe(function() {
    return gif(isLessFile, less({
		rootpath: "../"		
	}));
  });

module.exports = {
    bundle: {        
		main: {      
            styles: [
                __webapp + 'stylesheet/src/main.less'
            ],
            options: {
                useMin: false, // pre-minified files                 
                minCSS: true, 
                rev: true,
                transforms: {
                    styles: styleTransforms
                }
            }
        }
    }
};
```

Sample Gulp Task:
```javascript
//Bundle, compress, and write less to dist
gulp.task('lessMain', ['cleanCSS'], function() {  
    return gulp.src(projectPaths.cssMainConfig)        
        .pipe(bundle().on("error", gutil.log))        
        .pipe(bundle.results({
            fileName: 'main.result',
            dest: projectPaths.cssResults,
            pathPrefix: '<%=request.getContextPath()%>/dist/styles/'
        }))        
        .pipe(gulp.dest(projectPaths.cssDist));
});
```

**Comment by chmontgomery** [comment]
Thanks for writing up an issue.

I think the best way to fix this is to allow passing of plugin options to gulp-minify-css, as well as to all other plugins, e.g.:

```js
module.exports = {
    bundle: {        
        main: {      
            styles: [
                'stylesheet/src/main.less'
            ],
            options: {
                useMin: false, // pre-minified files                 
                minCSS: true, 
                rev: true,
                transforms: {
                    styles: styleTransforms
                },
                pluginOptions: {
                    'gulp-minify-css': {processImport: false},
                    'gulp-uglify': {mangle: false}
                }
            }
        }
    }
};
```

Thoughts? Do you have a better idea on how to structure this?

**Comment by smschrader** [comment]
No, I like this approach. Having control over the process being preformed would be ideal. It would be nice as well to add additional process, but I know that's out of scope of this issues.

I'm looking into how to incorporate gulp-remember into the current bundle flow.

**Comment by chmontgomery** [comment]
@smschrader if you have specific things you want to add to the flow that are not possible today, please create a new issue for that so I can track the work. Thanks!

**Comment by chmontgomery** [comment]
@smschrader we updated `gulp-minify-css` as well as adding options for plugins. Can you test `v2.20.0` and let me know if it works for you?

**Comment by smschrader** [comment]
Sorry for the delay, just got a chance to update and test, everything is working as expected.

**Comment by smschrader** [comment]
Actually, the error is fixed, but it's now not including any .less files when minCss is true. Even with the options passed.

So, the error being thrown has been addressed, but the underlying lying issue(s) is still there.

**Comment by smschrader** [comment]
So I actually got to spend time today investigating this issue. It appears your code is OK, and it's actually another bug within the clean-css plugin. There is an issue with how it "interprets" short hand css properties and converts their values. [Issue in Clean CSS](https://github.com/jakubpawlowicz/clean-css/issues/149)

For example in my case the shorthand 
    font: 11px/normal sans-serif;
got converted to 
    font: 11px/400 sans-serif;
when minify css was set to true which made it appear like the same issue.

**Comment by chmontgomery** [comment]
Thanks for looking into this. Do you know off the top of your head: will this problem be helped by upgrading to the latest `gulp-minify-css` version (1.2.0)?

**Comment by smschrader** [comment]
Let me go force the version and check.

**Comment by smschrader** [comment]
No, I updated to the most recent version of clean-css in your package and get the same error. I left a comment on their font thread as well.

Based on [this fixed](https://github.com/jakubpawlowicz/clean-css/commit/1c49d30da6821b73e0257e48f195825b94374f43) that was put in, it's still going to be an issue as it's only looking for "normal" and not checking boundaries before of after the string.

I'm sure it's almost a non issue and the value "normal" for line-height is rarely used. I inherited the code the was causing the issue and had to double check to see if it was even a legal value for line-height.
